README.md

```diff
@@ -10,3 +10,16 @@
 ## License
 
 Hyperledger Aries Framework Go is licensed under the [Apache License Version 2.0](LICENSE).
+
```

Comment:
This should go under proper sub-heading. Currently, it's inside Licence.

Reply:
added sub-heading